### ISSUE TITLE
Black Jack Ace function is bugged

### ISSUE THREAD

**Issue by Juderaw** [DankMemer/dank-memer-suggestions-and-bugs#1129]
If you start with the hand AA (this may be a problem with any hand with two aces that is still soft i have not noticed yet) AA is a soft 12. if you hit and get any card at a value of 10 the hand is a loss instead of the hard 12 (since aces are supposed to be 1 or 11 depending on what the hand needs) and i think this happens since it reads 22 before triggering the aces to fit the hand resulting in the loss.

tldr: Aces are 1 or 11, i had AA and hit and received a 10 and it resulted in 22 (loss) this is a bug because that should have been 12. 10+1+1 =12 (consistently happening.)

**Comment by ShadowEmpress** [comment]
![Screenshot_20200705-110131_Discord](https://user-images.githubusercontent.com/68754020/88427223-5f47e080-cda7-11ea-8ecd-fa2a2d355b94.jpg)

**Comment by samdabloxyccola** [comment]
You got a Q, so the Q will add 10 to your total

**Comment by andrewisaboomer** [comment]
should be 12, both aces become 1 and queen is 10 so it should be 12 not 22

**Comment by ShadowEmpress** [comment]
![Screenshot_20200705-110131_Discord](https://user-images.githubusercontent.com/68754020/88463244-43efda80-ce66-11ea-83b5-0b0d5d322509.jpg)

**Comment by ShadowEmpress** [comment]
![Screenshot_20200725-110109_Discord](https://user-images.githubusercontent.com/68754020/88463272-6681f380-ce66-11ea-9ddd-257994be1006.jpg)
